src/sssom_schema/schema/sssom_schema.yaml

```diff
@@ -436,10 +436,10 @@ slots:
       - value: https://www.wikidata.org/wiki/Q865360
         description: (the Wikidata identifier for the Jaccard index measure).
   issue_tracker_item:
-    description: A URL location of the issue tracker item discussing this mapping or mapping set.
-    range: uri
+    description: The issue tracker item discussing this mapping.
+    range: EntityReference
     examples:
-      - value: https://github.com/mapping-commons/sssom/issues/166
+      - value: SSSOM_GITHUB_ISSUE:166
```

Comment:
If you want to **allow** curification of the `issue_tracker_item` value without **promoting** that, you could leave the example to the full URL.

`EntityReference` is defined using LinkML’s `uriorcurie` type, so a full URL is always valid.

Reply:
I think this is ok for now; the uriorcurie base type is another black holes we are avoiding to think about right now, so lets move this issue to a future effort.